shelley/chain-and-ledger/executable-spec/src/TxData.hs

```diff
@@ -153,6 +153,14 @@ data DCert hashAlgo dsignAlgo
   | GenesisDelegate (GenKeyHash hashAlgo dsignAlgo, KeyHash hashAlgo dsignAlgo)
   deriving (Show, Eq)
 
+isRegKey :: DCert h d -> Bool
```

Comment:
the TxData is more related to Utxo, maybe this is better -> `shelley/chain-and-ledger/executable-spec/src/Delegation/Certificates.hs`

Reply:
@uroboros got it - I was looking at that module in addition to `TxData` but opted to define the predicates next to the datatype declarations.

Since there are other predicates in `Delegation.Certificates` that makes sense as a home though.